packages/ui/components/forms/Multiselect/index.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import MultiselectComponent from './multiselect'
+import { observer, u } from 'startupjs'
+
+import './index.styl'
+
+const Multiselect = ({
+  options,
```

Comment:
add support for `options` to be a simple array of strings, in which case it will use this string for both `label` and `value`. So if you receive `['red', 'green']` it will just be converted to [{ value: 'red', label: 'red' }, { value: 'green', label: 'green' }]

This is needed for parity with `Select` api which does support this.